### PR TITLE
fix(Dropdown): Support for configuring dropdown offset

### PR DESCRIPTION
**Pull request by cehsu**
The [redesigned EMR dashboard](https://app.asana.com/0/821652619613450/1104820821162262/f) requires the following properties of the dropdown component to be configurable: border-radius, border, offset.

Related CMS PR: https://github.com/sappira-inc/cms/pull/225

Tested locally via `yarn docs`
image

### CODE REVIEW

File: src/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ export default class Dropdown extends React.Component {
   }
 
   show = () => {
-    const { placement, boundariesElement } = this.props;
+    const { offset = '0, 10', placement, boundariesElement } = this.props;
```
Comment (choochootrain) [inline src/Dropdown/Dropdown.js]: add `offset` to `propTypes` and `defaultProps`?

File: src/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ export default class Dropdown extends React.Component {
     placement: 'bottom-start',
     boundariesElement: 'window',
     on: 'click',
+    border: true,
```
Comment (choochootrain) [inline src/Dropdown/Dropdown.js]: add default for `boxShadow`?

**Comment by kylealwyn** [comment]
Can we just configure the dropdown styles in cms `theme.js`?

**Comment by cehsu** [comment]
Thanks @kylealwyn, that's a better approach -- we can specify the styles in theme instead. An offset will still need to be passed down in order to specify how Popper.js will handle/calculate placement.

File: src/Dropdown/Dropdown.js (hunk 1 above)
Comment (cehsu) [reply src/Dropdown/Dropdown.js]: Thanks @choochootrain, great idea. Updated.

File: src/Dropdown/Dropdown.js [hunk 3]
```diff
@@ -82,7 +86,7 @@ export default class Dropdown extends React.Component {
           placement,
           modifiers: {
             offset: {
-              offset: '0, 10',
```
Comment (kylealwyn) [inline src/Dropdown/Dropdown.js]: Does this need to be `offset: this.props.offset`?

Comment (cehsu) [reply src/Dropdown/Dropdown.js]: Yes! That was the intention, thanks for catching the oversight, fixed.

Comment (kylealwyn) [reply src/Dropdown/Dropdown.js]: Was this tested locally? Browser would've thrown

Comment (cehsu) [reply src/Dropdown/Dropdown.js]: I hadn't tested it locally, is something like yalc the best tool for that? Or by running the docs command?

Comment (cehsu) [reply src/Dropdown/Dropdown.js]: Thanks for the reminder Kyle, screenshot added to description above. (I had been testing the changes locally by editing the molekule.es.js file in  cms/node_modules, and had forgotten to copy over the assignment.)

Comment (kylealwyn) [reply src/Dropdown/Dropdown.js]: Looks like you found `yarn docs` :)

**Pull request merged**